pkg/manifests/config_test.go

```diff
@@ -234,3 +235,79 @@ func TestHttpProxyConfig(t *testing.T) {
 		}
 	}
 }
+
+type fakePodsCapacity int
+
+func (fpc *fakePodsCapacity) PodsCapacity(context.Context) (int, error) {
+	return int(*fpc), nil
+}
+func TestEnforceBodySizeLimit(t *testing.T) {
+
+	mc_10 := fakePodsCapacity(10)
+	mc_1000 := fakePodsCapacity(1000)
+	for _, tt := range []struct {
+		name                string
+		config              string
+		expectBodySizeLimit string
+		expectError         bool
+		mockClient          *fakePodsCapacity
```

Comment:
we can use the interface type directly.

```suggestion
		pcr          PodsCapacityReader
```